website/docs/r/cluster.html.markdown

```diff
@@ -349,6 +349,11 @@ In addition to all arguments above, the following attributes are exported:
 * `mongo_uri_updated` - Lists when the connection string was last updated. The connection string changes, for example, if you change a replica set to a sharded cluster.
 * `mongo_uri_with_options` - connection string for connecting to the Atlas cluster. Includes the replicaSet, ssl, and authSource query parameters in the connection string with values appropriate for the cluster.
 * `connection_strings` - Set of connection strings that your applications use to connect to this cluster. More info in [Connection-strings](https://docs.mongodb.com/manual/reference/connection-string/). Use the parameters in this object to connect your applications to this cluster. To learn more about the formats of connection strings, see [Connection String Options](https://docs.atlas.mongodb.com/reference/faq/connection-changes/). NOTE: Atlas returns the contents of this object after the cluster is operational, not while it builds the cluster.
+
+Connection strings must be returned as a list, therefore to refer to a specific attribute value add index notation. Example: mongodbatlas_cluster.cluster-test.connection_strings.0.standard_srv
+
+Private connection strings may not be available immediately as the full connection may not have finalized by end of the Terraform run.  If the expected connection string(s) do not contain a value a terraform refresh may need to be performed to obtain the value. 
+
```

Comment:
we can specify that they can see the UI to verify if the connection is done in order to make `terraform refresh` or intent several times until the attributes are displayed, something like this.

Reply:
Some of those using the provider will be unable to also see the UI so I did not mention but agree, might make sense.   I'll update.

Reply:
Thanks 👍